data/sql/updates/pending_db_world/rev_1598238244990167657.sql

```diff
@@ -0,0 +1,28 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1598238244990167657');
+
+UPDATE `creature_template` SET `faction` = 21 WHERE `entry` = 14688;
+DELETE FROM `smart_scripts` WHERE `entryorguid` = 14688;
+INSERT INTO `smart_scripts` VALUES
```

Comment:
You are missing [...] (`entryorguid, [...]) VALUES

Reply:
updated the insert